homework.2/homework.2.playground/Contents.swift

```diff
@@ -0,0 +1,39 @@
+// Homework.2
+
+import Darwin
+
+let valueOne = 2.5
+let valueTwo = 9.7
+let valueThree = 6.9
+let valueFour = 8.2
+print("Original values - \(valueOne), \(valueTwo), \(valueThree), \(valueFour)\n")
+
+// Separating the integer part from values
+
+let valueOneWholePart = Int(valueOne)
+let valueTwoWholePart = Int(valueTwo)
+let valueThreeWholePart = Int(valueThree)
+let valueFourWholePart = round(valueFour)
+print("Whole parts - \(valueOneWholePart), \(valueTwoWholePart), \(valueThreeWholePart), \(Int(valueFourWholePart))\n")
+
+// Separating the fractional part from values
+
+let valueOneFractionalPart = valueOne.truncatingRemainder(dividingBy: 2)
+let valueTwoFractionalPart = valueTwo.truncatingRemainder(dividingBy: 3)
+let valueThreeFractionalPart = valueThree.truncatingRemainder(dividingBy: 2)
+let valueFourFractionalPart = valueFour.truncatingRemainder(dividingBy: 2)
```

Comment:
Why do you use 2 for some cases and 3 for one case? It's better to use the same number everywhere. Maybe just number 1 will be the best solution here?

Reply:
I foresaw this comment :) I thought about it already when I sent the assignment :) Thank you!

Reply:
Great that we think the same 😁 It's a good skill to foresee what questions your code can raise